src/main/java/com/leinardi/pycharm/mypy/MypyConfigService.java

```diff
@@ -31,11 +31,13 @@
     private String mypyConfigFilePath;
     private String mypyArguments;
     private boolean scanBeforeCheckin;
+    private boolean useDaemon;
 
     public MypyConfigService() {
         customMypyPath = "";
         mypyArguments = "";
         mypyConfigFilePath = "";
+        useDaemon = true;
```

Comment:
dmypy seems to be still somewhat experimental. In productive use we definitely encountered some unexpected errors here and there, as well as apparant infinite loops where dmypy would never finish. Therefore I would strongly suggest to disable this option by default.